src/common/SPELLS/thewarwithin/flasks.ts

```diff
@@ -0,0 +1,31 @@
+import Spell from '../Spell';
+
+const Flasks = {
+  FLASK_OF_ALCHEMICAL_CHAOS: {
+    id: 432021,
+    name: 'Flask of Alchemical Chaos',
+    icon: 'inv_potion_orange',
+  },
+  FLASK_OF_TEMPERED_AGGRESSION: {
+    id: 431971,
+    name: 'Flask of Alchemical Aggression',
+    icon: 'inv_potion_red',
+  },
+  FLASK_OF_TEMPERED_SWIFTNESS: {
+    id: 431972,
+    name: 'Flask of Alchemical Swiftness',
+    icon: 'inv_potion_green',
+  },
+  FLASK_OF_TEMPERED_VERSATILITY: {
+    id: 431973,
+    name: 'Flask of Alchemical Versatility',
+    icon: 'inv_potion_blue',
+  },
+  FLASK_OF_TEMPERED_MASTERY: {
+    id: 431974,
+    name: 'Flask of Alchemical Mastery',
+    icon: 'inv_potion_purple',
+  },
```

Comment:
While I doubt its used a lot, you did miss a flask: https://www.wowhead.com/spell=432473/flask-of-saving-graces

Reply:
Good catch! Added it too. Considered if it should be a "bad" flask that the module will warn about, but I figure it will be used in nicsche situations if at all 🤔 So I added it as a "good" flask